src/factories/Axis.ts

```diff
@@ -0,0 +1,89 @@
+module n3Charts.Factory {
+  'use strict';
+
+  export class Axis extends Utils.BaseFactory {
+
+    public svg: D3.Selection;
+    public scale: D3.Scale.Scale;
+    public axis: D3.Svg.Axis;
+
+    constructor(public name: string) {
+      super();
+
+    }
+
+    create() {
+      console.log('Create ' + this.key);
```

Comment:
If we wanna have a verbose mode, we could use `console.debug('Axis:: create ' + this.key);` or something... Otherwise we can't leave log statements in the code.